### PR TITLE
LPS-169786 | master

### PR DESCRIPTION
**Pull request by lilylcl**
https://issues.liferay.com/browse/LPS-169786

### CODE REVIEW

**Comment by lilylcl** [comment]
ci:test:sf

**Comment by lilylcl** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

File: portal-web/test/functional/com/liferay/portalweb/macros/User.macro [hunk 1]
```diff
@@ -1319,7 +1319,11 @@ definition {
 	macro editAddressCP {
 		UserNavigator.gotoContact();
 
-		LexiconEntry.gotoEllipsisMenuItem(menuItem = "Edit");
+		Click.waitForMenuToggleJSClick(locator1 = "Icon#VERTICAL_ELLIPSIS_BUTTON");
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/macros/User.macro:1322]: Please use the exist macro instead, you can find it via this locator you used. Thanks in advance.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @lilylcl,
It would be nice to use macro instead of functions here, when you find an exist locator that works for your case, please also using the locator to see if there are available macro as well. Thanks in advance.

**Comment by lilylcl** [comment]
Done and resend https://github.com/Sharryxue/liferay-portal/pull/512, thank you!

**Pull request closed without merging**